pkg/extensions/tbtc/derive_address_test.go

```diff
@@ -0,0 +1,98 @@
+package tbtc
```

Comment:
As the function we're testing is located in `liquidation_recovery.go` what do you think about naming this test file `liquidation_recovery_derive_address_test.go`? Or slicing `liquidation_recovery.go` to separate files? Maybe even a separate package for all liquidation recovery stuff?